src/odemis/acq/fastem_conf.py

```diff
@@ -91,6 +91,15 @@ def configure_scanner(scanner, mode):
     # Immersion needs to be set before changing the horizontalFoV, as the range is updated
     scanner.immersion.value = conf["immersion"]
 
+    if scanner.immersion.value is False:
```

Comment:
I think it'd simpler to read if you had `if scanner.immersion.value: ... else: ...`

Reply:
That should now just be: `if scanner.immersion.value` (no `is False`).